File: Praktikum 6/c/PRAK603_2410817110012_GtMuhammadLuthfiKamil.c
#include <stdio.h>

int main() {
    int n1, n2;

    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int matrix1[n1][1], matrix2[n2][1];

        for (int i = 0; i < n1; i++) {
            scanf("%d", &matrix1[i][0]);
        }

        for (int i = 0; i < n2; i++) {
            scanf("%d", &matrix2[i][0]);
        }

        for (int i = 0; i < n1; i++) {
            printf("%d ", matrix1[i][0] * matrix2[i][0]);
        }
        printf("\n");
    }

    return 0;
}

File: Praktikum 6/py/PRAK603_2410817110012_GtMuhammadLuthfiKamil.py
n1, n2 = map(int, input().split())

if n1 != n2:
    print("Jumlah tidak sama")
else:
    matrix1 = list(map(int, input().split()))

    matrix2 = list(map(int, input().split()))

    result = [matrix1[i] * matrix2[i] for i in range(n1)]
    print(*result)